App/app.js

```diff
@@ -0,0 +1,40 @@
+Ext.onReady(function(){
+    var viewport = new Ext.Viewport({
+        layout: 'border',
+        renderTo: Ext.getBody(),
+        items: [{
+            region: 'north',
+            xtype: 'panel',
+            html: 'North (Menus go here.)'
+        },{
+            region: 'west',
+            xtype: 'panel',
+            split: 'true',
+            width: 200,
+            html: 'West (Lists go here)'
+        },{
+            region: 'center',
+            xtype: 'panel',
+            items:[{
+                xtype: 'button',
+                text: 'Users',
+                cls: 'myButton'
+            },{
+                xtype: 'button',
+                text: 'Jobs',
+                cls: 'myButton'
+            },{
+                xtype: 'button',
+                text: 'Settings',
+                cls: 'myButton'
+
+            },{
+                xtype: 'button',
+                text: 'Lucky Guy!',
+                cls: 'myButton'
+            }]
+        }]
+    });
+
+
```

Comment:
Unnecessary empty row regarding our coding standards. Also noticed that the indention is space based and not tab based, so i guess you are not using phpStorm as IDE